packages/app/src/containers/Inspector.tsx

```diff
@@ -108,6 +109,9 @@ export default memo(function Inspector() {
       Layers.isBitmapLayer(l),
     );
     const hasTextLayer = selectedLayers.some((l) => Layers.isTextLayer(l));
+    const hasLineLayer = selectedLayers.some(
```

Comment:
We can allow editing multiple lines at once. It looks like Sketch show the LineInspector when every selected layer is a line. We can do this by changing `.some` to `.every`